kratos/tests/test_utilities/test_laplacian_element.cpp

```diff
@@ -20,7 +20,7 @@
 #include "includes/variables.h"
 #include "utilities/math_utils.h"
 #include "utilities/geometry_utilities.h"
-#include "tests/cpp_tests/auxiliar_files_for_cpp_unnitest/test_laplacian_element.h"
+#include "tests/test_utilities/test_laplacian_element.h"
```

Comment:
Can you move these files in an independent PR?

Reply:
I would prefer not to, but if its strictly necessary I can.

Reply:
This PR is humngous, it will simplify a lot review

Reply:
Ok I'll try, but this is only like the base of the PR. I will have to change all the apps as well. I will prepare smaller PR to merge to this one but there is no way that both systems coexists :S so changes need to be in a single PR